### ISSUE TITLE
Two Bug of NET5 WinForm Designer

### ISSUE THREAD

**Issue by Wildcatii** [dotnet/winforms#4219]
* .NET Core Version: 5.0

* Have you experienced this same bug with .NET Framework?: 

.NET Framework is normal.
**Problem description:**
1.Set image of button or label with designer cause NullException.
2.Can't access MenuStrip's (or ContextMenuStrip's)  item from designer directly.

**Expected behavior:**

**Minimal repro:**

**Comment by RussKie** [comment]
I appreciate you have spent the time to raise the issue, but issues with VS and VS Designer should be sent via VS Feedback channel (the icon in the top right hand side corner). The reason for this - the VS Feedback tool captures a lot of additional information (such as screenshots, memory dumps, stack traces, assembly versions, etc.) that can help us to investigate your issue and identify the cause.

**Comment by paul1956** [comment]
@RussKie There are many Feedback items going back over a year on these two items, most are closed as duplicate/not enough information/under investigation. I tried adding "Can't access MenuStrip's (or ContextMenuStrip's) item from designer directly" and there were over 4,000 feedback items that matched that string and the first several I checked did describe the issue fairly well, so if Feedback works this issue should have been marked as duplicate. Maybe there should be a GitHub Repo with Designer Issues similar to some of the Documentation Repo's. Then people can host code and images with enough detail to get these items addressed.
